File: java_project/Array/AverageOfElements.java
package Array;

public class AverageOfElements {
    public static void main(String[] args) {
        int a[] = new int[10];
        int highest = a[0];
        int lowest = a[0];
        int sum = 0;
        //int count = 0;
        for (int i = 0; i < a.length; i++) {
            a[i] = (i + 1);
            System.out.print(a[i] + "\t");
        }
        System.out.println("original sequence");
        for (int i = 0; i < a.length; i++) {
            sum = sum + a[i];
            if (a[i] > highest) {
                highest = a[i];
            }
            if (a[i] < lowest) {
                lowest = a[i];
            }

        }
        int filteredSum = sum - highest - lowest;
        int average = filteredSum / a.length - 2;
        System.out.println(sum + " is total sum");
        System.out.println(highest + " is highest");
        System.out.println(lowest + " is lowest");
        System.out.println(filteredSum + " is filtered sum");
        System.out.println(average + " is average");
    }
}

File: js_project/Arrays/AverageOfElements.js
let a= new Array(10).fill(0);
let highest = a[0];
let lowest = a[0];
let sum = 0;
for (let i=0; i<a.length; i++){
    a[i]=i+1;
    console.log(a[i]);
}
console.log("original Sequence");
for (let i=0; i<a.length; i++){
    sum = sum + a[i];
    if (a[i]>highest){
        highest = a[i];
    }
    if (a[i]<lowest){
        lowest=a[i];
    }

}
let filteredSum = sum- highest- lowest;
let average = filteredSum/a.length - 2;
console.log(sum," sum");
console.log(highest," highest");
console.log(lowest," lowest");
console.log(filteredSum);
console.log(average);